src/main/java/com/example/dr_app/service/ReservationService.java

```diff
@@ -39,4 +40,13 @@ public Reservation saveReservationData(Long doctorId, Long patientId) {
 
         return reservationRepository.save(reservation);
     }
+
+    public List<Reservation> getReservationsData() throws DataBaseException {
+
+        try {
+            return reservationRepository.findAll();
+        } catch (Exception e) {
+            throw new DataBaseException("Error occurred while fetching doctors data.", e);
```

Comment:
I guess this is a copy paste error, please fix the error message to the Reservation